coral/src/domain/topic/topic-api.ts

```diff
@@ -26,9 +26,12 @@ const getTopics = async (
   params: KlawApiRequestQueryParameters<"getTopics">
 ): Promise<TopicApiResponse> => {
   const queryParams = convertQueryValuesToString({
-    ...params,
```

Comment:
I removed that, since it passes all props and if there's no `teamId` set, it still has an empty teamId sent to the backend. It's not causing any errors, but it's not correct.